openmetadata-ui/src/main/resources/ui/playwright/utils/user.ts

```diff
@@ -196,13 +196,18 @@ export const hardDeleteUserProfilePage = async (
   await page.check('[data-testid="hard-delete"]');
   await page.fill('[data-testid="confirmation-text-input"]', 'DELETE');
 
+  const toastPromises = [
+    page.waitForSelector('[data-testid="alert-bar"]'),
+    page.waitForSelector('[data-testid="alert-icon"]'),
+    page.waitForSelector('[data-testid="alert-icon-close"]'),
+  ];
```

Comment:
not sure why we want to wait for all of this?

Reply:
Its not able to find the alert icons as it is getting closed and we are also redirecting the user to landing page.